node/lib/util/synthetic_gc_util.js

```diff
@@ -422,12 +424,13 @@ SyntheticGcUtil.prototype.populatePerCommit = co.wrap(
                 classAroots[subPath].add(subCommit);
                 this.d_subCommitStored[subPath][subCommit.sha()] = 1;
             } catch(exception) {
-                console.log("Cannot process submodule with following " +
-                    " exception: " + exception);
+                console.log("Cannot process submodule " + subName +
```

Comment:
we should probably at the very least:
1. log this to stderr and
2. exit with a non-zero status eventually, even if we do not do so immediately.